backend/pkg/tracing/init.go

```diff
@@ -0,0 +1,38 @@
+package tracing
+
+import (
+  "fmt"
+  "time"
+
+  log "github.com/golang/glog"
+
+  "github.com/uber/jaeger-client-go"
+  "github.com/uber/jaeger-client-go/config"
+  "github.com/opentracing/opentracing-go"
+)
+
+func Init(serviceName string, tracingClientHostPort string) opentracing.Tracer {
```

Comment:
Golang ma jazykovou ficuru `init` funkce. Tady je to vklidu, protoze mas `Init`. Pro info https://golang.org/doc/effective_go.html?#init